odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -50,27 +196,44 @@ def fbp_op(ray_trafo, padding=True):
         `CircularConeFlatGeometry` : Approximate reconstruction, correct in
         limit of fan angle = 0 and cone angle = 0.
 
-        `HelicalConeFlatGeometry` : Very approximate.
+        `HelicalConeFlatGeometry` : Very approximate unless a
+        `tam_danielson_window` is used. Accurate with the window.
```

Comment:
Perhaps @zickert could write more on this since you have been reading about Katsevich?